lib/reverse_words.rb

```diff
@@ -1,2 +1,8 @@
 def reverse_words(str)
+    x = str.split(" ")
+    y = ""
+    for i in 0..x.length-1
+        y = y + x.pop + " "
+    end
+    return y.slice(0, y.length-1)
```

Comment:
Hm. Nice fix, but I think it'd be more ideal if that trailing white space was not added to the value you want to return in the first place. Considering the data type of `y` might be helpful.